src/Core/Settings.h

```diff
@@ -842,6 +842,8 @@ class IColumn;
     M(Bool, print_pretty_type_names, false, "Print pretty type names in DESCRIBE query and toTypeName() function", 0) \
     M(Bool, create_table_empty_primary_key_by_default, false, "Allow to create *MergeTree tables with empty primary key when ORDER BY and PRIMARY KEY not specified", 0) \
     M(Bool, allow_named_collection_override_by_default, true, "Allow named collections' fields override by default.", 0)\
+    M(String, default_view_sql_security, "DEFINER", "Allows to set a default value for SQL SECURITY option when creating view.", 0) \
```

Comment:
It looks like `default_view_sql_security` should be a enum, not a string. There are only three possible values: `NONE`, `INVOKER`, `DEFINER`.